src/pages/Sidebar/Sidebar.tsx

```diff
@@ -21,49 +21,52 @@ export default function Sidebar({ element }: { element: React.ReactElement; }) {
     { title: "Settings", route: "/settings", icon: SettingsIcon },
   ];
 
-  const [selectedLink, setSelectedLink] = useState("Entries");
+  const [selectedLink, setSelectedLink] = useState("");
 
   const { height } = useWindowDimensions();
 
   return (
-    <div className="flex h-screen w-full">
-      <div className="scrollbar-y bottom-0 left-0 top-0 z-[60] hidden min-w-[320px] max-w-[320px] transform overflow-y-auto border-gray-200 bg-light_yellow transition-all duration-300 lg:bottom-0 lg:right-auto lg:block lg:translate-x-0">
+    <div className="flex w-full">
+      <div
+        id="docs-sidebar"
+        className="scrollbar-y scroll-hidden bottom-0 left-0 top-0 z-[60] h-screen min-w-[320px] max-w-[320px] transform overflow-y-auto border-gray-200 bg-light_yellow transition-all duration-300 lg:bottom-0 lg:right-auto lg:block lg:translate-x-0"
+      >
         <div className="flex h-full flex-col">
           <div className="m-4 flex grow flex-col">
             <div className="m-4 flex grow flex-col">
-              <div className="mb-1 text-sm font-semibold text-gray-600">
+              <div className="mb-1 text-sm font-semibold text-dark_gray">
                 Account
               </div>
-              {/*<a className="flex-none" href={"/profile/" + user!.id}>*/}
-              <div className="mb-6 flex items-center rounded-xl">
-                <div
-                  className="mr-2 items-center"
-                  onClick={() => setSelectedLink("")}
-                >
-                  <img
-                    className="mx-auto block max-h-[40px] max-w-[40px] justify-center rounded-full sm:mx-0 sm:shrink-0"
-                    src={user!.profile_picture}
-                    alt="profile"
-                  />
-                </div>
-                <div className="mr-2 text-secondary">
-                  <div className="text-base font-semibold">
-                    {user!.username}
+              <a className="flex-none" href={"/profile/" + user!.id}>
```

Comment:
adding the <a> breaks the logout button. also design changed the way this works to a dropdown right?

Reply:
added dropdown so this should be good, and logout seems to work